Tools/Gulp/gulp-addModuleExports.js

```diff
@@ -18,11 +18,11 @@ module.exports = function (varName) {
         '};\n';
 
         var decorateAddition =
-        'var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {\n' +
```

Comment:
Sorry for this - my editor removed the windows carriage return - do you want me to revert that change? It seemed to be inconsistently used in the repo.